### ISSUE TITLE
Support for the X3D (Extensible 3D) standard

### ISSUE THREAD

**Issue by vault-thirteen** [FreeCAD/FreeCAD#18265]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

The editor lacks support for the X3D (Extensible 3D) standard.

19775

> ISO/IEC 19775-1:2023
> https://www.iso.org/standard/82562.html
> 
> ISO/IEC 19775-2:2015
> https://www.iso.org/standard/60500.html

19776

> ISO/IEC 19776-1:2015
> https://www.iso.org/standard/60502.html
> 
> ISO/IEC 19776-2:2015
> https://www.iso.org/standard/60503.html
> 
> ISO/IEC 19776-3:2015
> https://www.iso.org/standard/60504.html

19777

> ISO/IEC 19777-1:2006
> https://www.iso.org/ru/standard/33915.html
> 
> ISO/IEC 19777-2:2006
> https://www.iso.org/ru/standard/38020.html

### Subproject(s) affected?

Core

### Anything else?

X3D is a unfied standard for 3D computer graphics. 
It is also known as an international (i.e. world-wide) standard ISO/IEC 19775/19776/19777.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
Please provide links to standard.

**Comment by vault-thirteen** [comment]
ISO/IEC 19775-1:2023
https://www.iso.org/standard/82562.html

ISO/IEC 19776-1:2015
https://www.iso.org/standard/60502.html

ISO/IEC 19777-1:2006
https://www.iso.org/ru/standard/33915.html

**Comment by luzpaz** [comment]
Are those standards behind a paywall?

**Comment by FlachyJoe** [comment]
Some info here https://www.web3d.org/specifications/

**Comment by vault-thirteen** [comment]
> Are those standards behind a paywall?

No, no. Most of the documents are free to download, and other part of documents are available for zero cost.

**Comment by maxwxyz** [comment]
Same as https://github.com/FreeCAD/FreeCAD/issues/7268 ?
